tsconfig.dev.json

```diff
@@ -27,10 +27,12 @@
   "include": [
     ".projenrc.js",
     "lib/**/*.ts",
-    "test/**/*.ts"
+    "test/**/*.ts",
+    "assets/**/*.ts"
```

Comment:
We actually need the jsii-generated `tsconfig.json` to be changed, not this one.

I did figure out how to do that, but I'm experimenting with just using vanilla JS for the Lambda handlers.